lib/graphql-error-handler.js

```diff
@@ -0,0 +1,17 @@
+let graphqlErrorHandler = null;
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
+  const newrelic = require('newrelic');
+  graphqlErrorHandler = error => {
+    newrelic.noticeError(error, { graphql: true });
```

Comment:
Wouldn’t that mask the name in the error message https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name ?